src/logs/metadata_engine/metadata_engine.py

```diff
@@ -227,7 +227,7 @@ def _create_config_rule(entity_name: str, rule_json: Dict) -> Optional[ConfigRul
         aws_loggroup_pattern = rule_json["aws"]["logGroup"]
     except KeyError:
         aws_loggroup_pattern = None
-    log_content_parse_type = rule_json.get("aws", {}).get("logContentParseAs", None)
+    log_content_parse_type = rule_json.get("aws", {}).get("logContentParseAs", "")
```

Comment:
Why this change, given that you are checking this field with "==" ?

Reply:
![image](https://user-images.githubusercontent.com/50921490/122407385-e256c400-cf81-11eb-9df2-7b26da704f2b.png)
Hmm, it made it harder for me to make local debugging changes of the sort of:
```
record["log_content_parsed"]["requestParameters"] = {
        "dBInstanceIdentifier": "belu-metadata-database-1"
    }
```
as log_content_parsed wasn't there. 

Not sure if in python check against None or empty string makes a lot of difference. I'd say not and I'd leave it, but if you want it back, I won't insist.

Reply:
Must have appeared in the process of switching to ==

Reply:
tested & reverted